src/main/java/org/skriptlang/skript/bukkit/itemcomponents/equippable/elements/CondEquipCompDispensable.java

```diff
@@ -0,0 +1,66 @@
+package org.skriptlang.skript.bukkit.itemcomponents.equippable.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableExperiment;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableWrapper;
+
+@Name("Equippable Component - Is Dispensable")
+@Description("Whether an item can be dispensed by a dispenser. "
+	+ "Note that equippable component elements are experimental making them subject to change and may not work as intended.")
+@Example("""
+	if {_item} is dispensable:
+		add "Dispensable" to lore of {_item}
+	""")
+@Example("""
+	set {_component} to the equippable component of {_item}
+	if {_component} is not dispensable:
+		make {_component} dispensable
+	""")
+@RequiredPlugins("Minecraft 1.21.2+")
+@Since("INSERT VERSION")
+public class CondEquipCompDispensable extends PropertyCondition<EquippableWrapper> implements EquippableExperiment {
+
+	static {
+		Skript.registerCondition(CondEquipCompDispensable.class, ConditionType.PROPERTY,
+			"%equippablecomponents% (is|are) [:un]dispensable",
+			"%equippablecomponents% (isn't|is not|aren't|are not) [:un]dispensable"
```

Comment:
dispensable doesn't really mean `able to be dispensed` is the context of a dispenser. it means, like, worthless, discardable, superfluous. This should be something like `can be dispensed`.